src/tests/data/raidsData.ts

```diff
@@ -0,0 +1,328 @@
+          avatarUrl:
+            'https://avatars1.githubusercontent.com/u/22864071?u=311c5d3967f30c5f00f1b2cf3ae9952807e7a23a&v=4',
+          deletions: 32,
+          userId: 22864071,
+          commits: 2,
+          user: 'tsuki42',
+        },
+        23029903: {
+          additions: 253,
+          avatarUrl:
+            'https://avatars2.githubusercontent.com/u/23029903?u=86ef88261fe287ecea4c5d678c11abb4e8d40222&v=4',
+          deletions: 378,
+          userId: 23029903,
+          commits: 10,
+          user: 'mpeyper',
+        },
+        25822851: {
+          additions: 231,
+          avatarUrl:
+            'https://avatars0.githubusercontent.com/u/25822851?u=d328d59b4d75cb084266e6f7141dbcd2cc9c5b94&v=4',
+          deletions: 180,
+          userId: 25822851,
+          commits: 7,
+          user: 'kiranjd',
+        },
+        27247160: {
+          additions: 1901,
+          avatarUrl:
+            'https://avatars3.githubusercontent.com/u/27247160?u=768c4a975a73af6f5111c2af628c10939f7d0f47&v=4',
+          deletions: 2913,
+          userId: 27247160,
+          commits: 55,
+          user: 'JacobMGEvans',
+        },
+        35306025: {
+          additions: 2163,
+          avatarUrl:
+            'https://avatars2.githubusercontent.com/u/35306025?u=abc450125ac88a3eb77a5ae7c374258f2e43d3d4&v=4',
+          deletions: 2858,
+          userId: 35306025,
+          commits: 72,
+          user: 'codyarose',
+        },
+        40751395: {
+          additions: 2305,
+          avatarUrl:
+            'https://avatars2.githubusercontent.com/u/40751395?u=dbb71f044fe955ecb0e59047036dba2ca32c824d&v=4',
+          deletions: 2994,
+          userId: 40751395,
+          commits: 87,
+          user: 'nobrayner',
+        },
+        49603590: {
+          additions: 464,
+          avatarUrl:
+            'https://avatars3.githubusercontent.com/u/49603590?u=011adf63409359a3a222a718f5b8230e1dcecc79&v=4',
+          deletions: 539,
+          userId: 49603590,
+          commits: 15,
+          user: 'tigerabrodi',
+        },
+        60733989: {
+          additions: 1521,
+          avatarUrl:
+            'https://avatars1.githubusercontent.com/u/60733989?u=5f928c457d9e67074fec9dae5bca1de0d8dffd42&v=4',
+          deletions: 1672,
+          userId: 60733989,
+          commits: 43,
+          user: 'emma-r-slight',
+        },
+        73331245: {
+          additions: 6,
+          avatarUrl: 'https://avatars1.githubusercontent.com/u/73331245?v=4',
+          deletions: 7,
+          userId: 73331245,
+          commits: 1,
+          user: 'accentsoftware-braydonh',
+        },
+      },
+      status: 'active',
+    },
+  ],
+  error: null,
+}
+const fetchedDocumentData = {
+  state: 'success',
+  data: {
+    additions: 5719,
+    deletions: 8506,
+    title: 'Migrate to React Testing Library',
+    dungeon: 'HospitalRun/hospitalrun-frontend',
+    changedFiles: 169,
+    commits: 568,
+    contributors: {
```

Comment:
Similarly, reduce the number of contributors in this test data to 3 - that's all that is needed to test the sorting